packages/react-app/pages/admin/batches.jsx

```diff
@@ -191,11 +191,7 @@ export default function BuilderListView({ serverUrl, mainnetProvider, userRole }
           accessor: "lastActivity",
           sortDescFirst: true,
           disableFilters: true,
-          Cell: ({ value }) => (
-            <Text whiteSpace="nowrap">
-              <DateWithTooltip timestamp={value} />
-            </Text>
-          ),
+          Cell: ({ value }) => <DateWithTooltip timestamp={value} />,
```

Comment:
Haven't tested this yet, but you are removing the `nowrap` thing. 

If it was there, I have to think that it was for a reason (probably because it was wrapping to another line in some context), so I'd try not to remove it.

Probably we can just use a div (or <Box>) and the warning will be gone too.

This reminded me of this video haha https://www.youtube.com/watch?v=iIPJrWXgmxU

Reply:
Haha you are right , added ```nowrap``` in ```<Box>``` back